# Issue: Invalid mode error when trying to update files

Description: When trying to update files using the files.py module, an 'invalid mode' error is returned. This happens regardless of the mode specified. The issue seems to be in the update_file function in the files.py module.